### DISCUSSION TITLE
Build Failed

### DISCUSSION THREAD

**Discussion by MeganathanTraxx** [vercel/vercel#8932] [Help] [answered]
Hi

Our latest build got failed on Vercel, but it's working fine with our local I have noticed one thing the failed build Vercel CLI version is 28.5.3 and the last (yesterday) successful build Vercel CLI version is 28.4.17. Our node version is 14x and we are using turbo. I have attached the screenshots below

Failed build screenshot
![image](https://user-images.githubusercontent.com/100118001/202443186-18a1b52e-43ea-4b9c-a1b0-857b853a5dd1.png)

Success build screenshot
![image](https://user-images.githubusercontent.com/100118001/202443351-e2018175-37c9-41c2-82e0-24870d5af8c0.png)

Please let me your inputs regarding this. Thanks in advance.

**Comment by MeganathanTraxx** [comment]
I have fixed the issue using this reference [link](https://github.com/vercel/vercel/discussions/8908#discussioncomment-4143387)

Please share your inputs regarding this.

**Reply by amyegan** [reply]
I'm glad you found that workaround! A problem in Vercel CLI 28.5.3 was identified and the fix will be released today 🙇

**Reply by amyegan** [reply]
The build container was updated to use Vercel CLI 28.5.5. Please clear the `VERCEL_CLI_VERSION` so you don't get stuck on an old version, and let me know if you see any more issues with your builds 🙇

**Comment by MeganathanTraxx** [comment]
Thanks for your confirmation.

**Comment by amyegan** [comment, marked as answer]
This issue has been resolved. If you still have `VERCEL_CLI_VERSION` set as a workaround, please remember to remove it now so you don't get stuck on an old CLI version 🙏

**Reply by MeganathanTraxx** [reply]
hey @amyegan 
thanks for your assist, it's working fine with the latest CLI version.
